### PR TITLE
fox dist err

### PR DESCRIPTION
**Pull request by littletomatodonkey**
fix some errors by tingquan

### CODE REVIEW

File: ppcls/arch/__init__.py [hunk 1]
```diff
@@ -82,13 +81,13 @@ def forward(self, x, label=None):
...
         if self.neck is not None:
             x = self.neck(x)
+            out["neck"] = x
         out["features"] = x
         if self.head is not None:
             y = self.head(x, label)
-            out["neck"] = x
+            out["logits"] = x
         else:
             y = None
```
Comment (TingquanGao) [inline ppcls/arch/__init__.py]: out["logits"] = y

**Review by TingquanGao — APPROVED** [review]
